### ISSUE TITLE
CLI prints warning for non-existent export 'default'

### ISSUE THREAD

**Issue by runspired** [emberjs/data#6630]
```
Non-existent export 'default' is imported from -private/ts-interfaces/record-data.js
```

This is a typescript file and it does have a default export, I suspect the issue is rollup encountering the file after types have been stripped. Changing these type imports to `import()` syntax may resolve this.

**Comment by Gaurav0** [comment]
Looks to me like this is an instance of https://github.com/babel/babel/issues/8361

**Comment by runspired** [comment]
Resolved in #6809
